subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/ConsoleGradleBuildGroupedTaskFunctionalTest.groovy

```diff
@@ -14,19 +14,18 @@
  * limitations under the License.
  */
 
-package org.gradle.internal.logging
+package org.gradle.internal.logging.console.taskgrouping
 
-import org.gradle.integtests.fixtures.AbstractConsoleFunctionalSpec
-import spock.lang.Ignore
+import spock.lang.FailsWith
 
-@Ignore("Build fails and passes depending on timing")
-class ConsoleGradleBuildFunctionalTest extends AbstractConsoleFunctionalSpec {
+class ConsoleGradleBuildGroupedTaskFunctionalTest extends AbstractConsoleGroupedTaskFunctionalTest {
 
     private static final String HELLO_WORLD_MESSAGE = 'Hello world'
     private static final String IMPORTANT_MESSAGE = 'Something important needs to happen'
     private static final String BYE_WORLD_MESSAGE = 'Bye world'
     private static final String AGGREGATE_TASK_NAME = 'all'
 
+    @FailsWith(value = AssertionError, reason = "External build does not group task output")
```

Comment:
We don't really have an issue for it at the moment. I believe it's simply not working yet. I might create an issue though I don't think we'll do something about it anytime soon.